scipy/sparse/linalg/_eigen/_svds.py

```diff
@@ -90,15 +90,16 @@ def _iv(A, k, ncv, tol, which, v0, maxiter,
     if return_singular not in rs_options:
         raise ValueError(f"`return_singular_vectors` must be in {rs_options}.")
 
-    random_state = check_random_state(random_state)
+    rng = check_random_state(rng)
 
     return (A, k, ncv, tol, which, v0, maxiter,
-            return_singular, solver, random_state)
+            return_singular, solver, rng)
 
 
+@_transition_to_rng("random_state")
```

Comment:
Hopefully my counting is correct 
```suggestion
@_transition_to_rng("random_state", position_num=9)
```